examples/maxq10xx-wolfssl-client.c

```diff
@@ -0,0 +1,335 @@
+/* maxq10xx-wolfssl-client.c
```

Comment:
For something specific to a particular piece of hardware like this, I wonder if the examples repo is a better location.

Reply:
ah, Yes. I think so as well. This was put here temporarily so the customer wouldn't have to look in the examples. repo.  This will be removed before final merge.

Reply:
Done with 7d04d275f